tools/format.py

```diff
@@ -21,24 +21,34 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 #####################################################################################
-import os, shutil, argparse, subprocess
+import os, shlex, shutil, argparse, subprocess
 
 CLANG_FORMAT_PATH = '/opt/rocm/llvm/bin'
 
+EXCLUDE_FILES = ['requirements.in']
```

Comment:
So this doesn't need a path?  The only requirements.in I see in our code base is [docs/sphinx/requirements.in](https://github.com/ROCm/AMDMIGraphX/blob/a2752d8e9e7e0c959a0506988938ec125d048f3e/docs/sphinx/requirements.in)

Reply:
> So this doesn't need a path?

Nope, it checks the basename.

Reply:
I see, there is a mistake there.

Reply:
Fixed.